app/static/js/request/main.js

```diff
@@ -48,6 +48,15 @@ function characterCounter(target, limit, currentLength, minLength) {
     }
 }
 
+function generate_character_counter_handler(id, length) {
+    /*
+     * This function creates character counter handlers for when you have multiple events inside a loop
+     */
+    return function () {
+        characterCounter("#" + id + "-character-count", length, $(this).val().length);
```

Comment:
This is specifically taking in maxLength, right? Can't this function also take minLength?